packages/databases-collections/src/components/collection-fields/collection-fields.jsx

```diff
@@ -149,17 +166,35 @@ export default class CollectionFields extends PureComponent {
         <TimeSeriesFields
           isCapped={isCapped}
           isTimeSeries={isTimeSeries}
+          isClustered={isClustered}
           onChangeIsTimeSeries={(newIsTimeSeries) => this.setState(
             { isTimeSeries: newIsTimeSeries },
             this.updateOptions
           )}
-          onChangeTimeSeriesField={(fieldName, value) =>
+          onChangeField={(fieldName, value) =>
             this.setField(fieldName, value)
           }
           timeSeries={timeSeries}
           expireAfterSeconds={expireAfterSeconds}
         />
       )}
+      {hasClusteredCollectionSupport(serverVersion) && (
+        <ClusteredCollectionFields
+          isCapped={isCapped}
+          isTimeSeries={isTimeSeries}
+          isClustered={isClustered}
+          clusteredIndex={clusteredIndex}
+          expireAfterSeconds={expireAfterSeconds}
+          onChangeIsClustered={(newIsClustered) => this.setState(
+            { isClustered: newIsClustered },
```

Comment:
Hmm good point. I just copied this blindly from timeseries code. Should probably clear it there as well.

Reply:
That expireAfterSeconds property is weird because it is kinda a collection option, not necessarily timeseries or clustered collections specific. But it is only becoming an option in conjunction with those. I think eventually you'll be able to put it on any collection.